GameData/ROCapsules/PartConfigs/Apollo D2/D2Solar.cfg

```diff
@@ -0,0 +1,88 @@
+PART
+{
+	name = ROC-D2Solar
+	module = Part
+	author = Mcdouble, Vader111
+
+	RSSROConfig = True
+
+	//	============================================================================
+	//	Model and Dimensions
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROCapsules/Assets/McDouble/D2/D2Solar
+	}
+
+	scale = 1
+	rescaleFactor = 1.0
+
+	node_attach = 0.0, 0.0, 0.0, 0.0, 1.0, 0.0
+
+	attachRules = 0,1,0,0,1
+
+	//	============================================================================
+	//	Title, Description, Category, Techs
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = Electrical
+	subcategory = 0
+
+	title = D-2 Solar Array
+	manufacturer = General Electric
+	description = Deployable solar arrays for extended orbital missions.
+	tags = D2, apollo, GE
+
+	bulkheadProfiles = srf
+
+	//	============================================================================
+	//	Thermo, Durability
+	//	============================================================================
+
+	mass = 0.02
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 1
+	crashTolerance = 10
+	breakingForce = 250
+	breakingTorque = 250
+	maxTemp = 673.15
+	skinMaxTemp = 773.15
+	thermalMassModifier = 2.0
+	emissiveConstant = 0.95
+	heatConductivity = 0.04 // 1/3 the default
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = ModuleDeployableSolarPanel
+		resourceName = ElectricCharge
+		chargeRate = 0.420
+		animationName = deployArray
+		raycastTransformName = suncatcher
+		pivotName = suncatcher
+		isBreakable = true
+		isTracking = false
+		extendActionName = Extend
+		retractActionName = Retract
+		extendpanelsActionName = Toggle
+	}
+
+	useSolarConfig = true
+	solarUseMass = true
+	
+	SolarConfig
+	{
+		area = 2.2868
+		level = 2
+		type = tracking
```

Comment:
Solar level 2 might be a little low there.  Confirm these stats would give you the rated output power you want?  (This will replace the 420W value in the MDSP PartModule)

Reply:
level 2 is 1967 tech which is around the time the solar panels would be started to be used

Reply:
That is true of the tech (although we may be rebalancing that), but DRVeyl's point stands that the panels at their current size at level2 are highly unlikely to be able to power the CSM.